src/components/create-card.jsx

```diff
@@ -1,7 +1,62 @@
 import React from "react";
 
-function CreateCard(props) {
-  return <h1 className="text-center">Create New Card</h1>
+class CreateCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      question: "",
+      answer: ""
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
+
+  handleChange(event) {
+    const target = event.target;
+    if (target.name === "question") {
+      this.setState({
+        question: target.value
+      })
+    } else if (target.name === "answer") {
+      this.setState({
+        answer: target.value
+      })
+    }
+  }
```

Comment:
Notice here how `target.name` is the same as the key in state... You can use that to simplify this function and make it more versatile if you add more inputs in the future:

```JAVASCRIPT
handleChange(event) {
  const { name, value } = event.target;
  
  this.setState({
    [name]: value
  });  
}
```